main.go

```diff
@@ -90,6 +103,15 @@ It can be used in a curl-like fashion when benchmarking features are disabled.
 		return opts, errExit
 	}
 
+	// Read defaults from ~/.yab.ini
+	if user, err := user.Current(); err != nil {
```

Comment:
hmm true, let's leave this as is then, i think there's other places using `$USER`, we should probably move all the user-specific logic into a single place in future.